ksonnet-gen/kubeversion/version.go

```diff
@@ -26,6 +26,16 @@ func KSource(k8sVersion string) string {
 	return verData.kSource
 }
 
+// Beta returns the beta status of the version.
+func Beta(k8sVersion string) bool {
+	verData, ok := versions[k8sVersion]
+	if !ok {
```

Comment:
We probably don't want to be erroring out on if someone supplies `1.8.1` or `1.7.5`. Thoughts on how this should be handled? I think we should check for patch versions and as long as the minor version is supported, return true / false accordingly. Only error where minor versions aren't supported.

Likewise is `1.6` currently supported by `ks`? If yes, we should probably return false for anything less than the highest minor version.

Reply:
I've updated it to return false if 1) it doesn't know the version, 2) the version beta flag is set to false.

Reply:
Think we may need to do a little more work here because `ks` wouldn't be able to warn users on `1.8.1`.

Maybe for now just add a check:

```
const (
  maxVersion := "v.1.8."
)

if strings.HasPrefix(k8sVersion, maxVersion)  {
  return true
}
```

or something.